src/Playwright.Tests/SelectorMiscTests.cs

```diff
@@ -99,102 +99,51 @@ public async Task ShouldWorkWithLayoutSelectors()
-            Assert.AreEqual(await Page.EvalOnSelectorAsync<string>("div >> below=\"#id6\"", "e => e.id"), "id8");
             Assert.AreEqual(await Page.EvalOnSelectorAsync<string>("div:below(#id7)", "e => e.id"), "id8");
-            Assert.AreEqual(await Page.EvalOnSelectorAsync<string>("div >> below=\"#id7\"", "e => e.id"), "id8");
             Assert.AreEqual(await Page.EvalOnSelectorAsync<string>("div:below(#id8)", "e => e.id"), "id9");
-            Assert.AreEqual(await Page.EvalOnSelectorAsync<string>("div >> below=\"#id8\"", "e => e.id"), "id9");
             Assert.AreEqual(await Page.QuerySelectorAsync("div:below(#id9)"), null);
-            Assert.AreEqual(await Page.QuerySelectorAsync("div >> below=\"#id9\""), null);
             Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div:below(#id3)", "els => els.map(e => e.id).join(',')"), "id0,id5,id6,id7,id8,id9");
-            Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div >> below=\"#id3\"", "els => els.map(e => e.id).join(',')"), "id0,id5,id6,id7,id8,id9");
             Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div:below(#id3, 105)", "els => els.map(e => e.id).join(',')"), "id0,id5,id6,id7");
-            Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div >> below=\"#id3\" , 105", "els => els.map(e => e.id).join(',')"), "id0,id5,id6,id7");
 
             Assert.AreEqual(await Page.EvalOnSelectorAsync<string>("div:near(#id0)", "e => e.id"), "id3");
-            Assert.AreEqual(await Page.EvalOnSelectorAsync<string>("div >> near=\"#id0\"", "e => e.id"), "id3");
-            Assert.AreEqual(await Page.Locator("div", new() { Near = Page.Locator("#id0") }).First.EvaluateAsync<string>("e => e.id"), "id3");
             Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div:near(#id7)", "els => els.map(e => e.id).join(',')"), "id0,id5,id3,id6");
-            Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div >> near=\"#id7\"", "els => els.map(e => e.id).join(',')"), "id0,id5,id3,id6");
             Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div:near(#id0)", "els => els.map(e => e.id).join(',')"), "id3,id6,id7,id8,id1,id5");
-            Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div >> near=\"#id0\"", "els => els.map(e => e.id).join(',')"), "id3,id6,id7,id8,id1,id5");
             Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div:near(#id6)", "els => els.map(e => e.id).join(',')"), "id0,id3,id7");
-            Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div >> near=\"#id6\"", "els => els.map(e => e.id).join(',')"), "id0,id3,id7");
             Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div:near(#id6, 10)", "els => els.map(e => e.id).join(',')"), "id0");
-            Assert.AreEqual(await Page.EvalOnSelectorAllAsync<string>("div >> near=\"#id6\",10", "els => els.map(e => e.id).join(',')"), "id0");
```

Comment:
When porting next time, it is not necessary to port all string selector tests, since for the port a string is just a string after all :)